workflow-templates/im-test-sam-yaml.yml

```diff
@@ -0,0 +1,52 @@
+# Workflow Code: DigitalWerewolf_v1    DO NOT REMOVE
+# Purpose:
+#    Runs the SAM YAML Validation action and then imports the
+#    output JSON into the Stewardship Mapping document repository
+#
+# Frequency:
+#    - This workflow should only be used once per repository
+#
+# Projects to use this Template with:
+#    -  Any project that needs to report SAM information for Stewardship Mapping
+#       See https://github.com/im-practices/master-architecture-plan/blob/main/Implementation%20Decisions/adr-015_system_architecture_metadata.md
+#       for more information
+#
+# TODO: Prerequisites:
+#    - Create a sam.yaml file that describes the Bounded Context projet information
+
+name: SAM.yaml Test and Import to Stewardship Mapping
+
+on:
+  # After a PR is merged, the workflow will check for changes to the SAM YAML file in the root directory
+  pull_request:
+    types: [closed]
+    paths:
+      - 'sam.yaml' # TODO - Verify sam.yaml file name and path. This path is for the a sam.yaml file located at the root of the proejct
+
+jobs:
+  sam-yaml:
+    name: Test and Import SAM YAML
+    if: github.event.pull_request.merged == true && github.base_ref == 'main'
+
+    runs-on: [ubuntu-latest]
+
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v2
+
+      - name: Validate SAM.yaml file
+        id: parse-yaml
+        uses: 'im-open/yaml-file-validation@v1.1.0'
+        with:
+          yaml-file-path: './sam.yaml' #TOOD - verify sam.yaml file name and path, it's different than the trigger filter path
```

Comment:
```suggestion
          yaml-file-path: './sam.yaml' #TODO - verify sam.yaml file name, extension and path, this might be slightly different than the trigger filter because it requires a relative path.
```

Reply:
👍🏽